doc/tutorials/docbuild.rst

```diff
@@ -140,7 +139,7 @@ For Ubuntu use:
 
   .. code-block:: bash
 
-     sudo apt install doxygen python3-pip \
+     sudo apt install doxygen=1.8.13-10 python3-pip \
```

Comment:
The problem with this approach is that it only works on Ubuntu 18.04. That version is not available on Ubuntu 20.04 for example. That may be the case too for other distros (e.g. Fedora, OpenSUSE Tumbleweed).

I think we need to be more explicit that it is indeed the version that is known to work, *but* that this exact version is only available on Ubuntu 18.04 and we recommend that people try first with the `doxygen` version that comes with their distro, even if the version does not exactly match.

Reply:
OK, sigh.  Thanks for catching that.